pipeline/terraform/integration-test-environment/lbs.tf

```diff
@@ -0,0 +1,112 @@
+resource "aws_lb" "outbound_alb" {
+  internal = true
+  load_balancer_type = "application"
+  subnets = aws_subnet.mhs_subnet.*.id
+  security_groups = [
+    aws_security_group.alb_outbound_security_group.id
+  ]
+
+  tags = {
+    Name = "${var.environment_id}-mhs-outbound-alb"
+    EnvironmentId = var.environment_id
+  }
+}
+
+resource "aws_lb_target_group" "outbound_alb_target_group" {
+  port = 80
+  protocol = "HTTP"
+  target_type = "ip"
+  vpc_id = aws_vpc.mhs_vpc.id
+
+  health_check {
+    path = "/"
+    matcher = "405"
```

Comment:
The application load balancers do these health checks. Currently, they're doing them on the same endpoint that we make requests to MHS to. It would be cleaner if there were separate healthcheck endpoints that just respond with HTTP 200 without doing any further processing. I think we should maybe have a separate task/techdebt to do this.